app/views/questions/show.html.erb

```diff
@@ -12,6 +12,9 @@
         <br>
         <div id="question-body" >
           <%= @question.body %><br>
+          <div id="edit-question">
```

Comment:
That restriction will need to be in place. I think that logic should live in a conditional in the html along the lines of 'if question.user = current_user" and display the edit button.